### PR TITLE
Add Form component and App refactor

### LINKED ISSUE #6: As a user, I can take a sentiment quiz
- [x] Add sentiment form w/ emojis and submit button
- [x] Route to home page on submit
- [x] Get articles from NYT
- [x] Isolate and clean relevant data

Acceptance Criteria
- Scenario: As a user, I'm feeling negative/neutral/positive and want to read the news
- Given: A sentiment quiz
- When I select my sentiment and click the submit button
- Then I'm taken to the feed and see the top articles

### PR DESCRIPTION
**Pull request by najinl**
closes #6 
### Features 
- Add Form component and associated CSS

### Refactors 
- Minor refactor in App to import Form and pass props to Form.

### Issues and Technical Debt
- We added a tertiary to toggle between Form and Feed to ensure they are mounting, we will need to utilize Router in the future.
- We will need to add a componentDidMount() to populate the state.

### Other Comments, Concerns, or Questions
- NA

### Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -8,10 +8,12 @@ interface PropsCard {
 
 const Card: React.FC<PropsCard> = ({ title, image }) => {
```
Comment (nzambonivergara) [inline src/components/Card/Card.tsx:9]: Should we remove the React.FC from our functional components now and follow [this example](https://github.com/typescript-cheatsheets/react/blob/main/README.md#function-components)?

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -56,17 +59,24 @@ class App extends React.Component<Props, State> {
...
       <div className="App">
-        <header className="App-header">
-          <h1>WellNews</h1>
-        </header>
-        <Feed articles={this.state.articles} />
+        <div className="app-container">
+          <header className="App-header">
+            <h1 className="header-text">WellNews</h1>
+          </header>
+          {this.state.initialSentimentSet ? <Feed articles={this.state.articles} /> : <Form userSentiment={this.changeUserSentiment}/>}
```
Comment (nzambonivergara) [inline src/components/App/App.tsx:78]: I would keep the name of the Form's prop of userSentiment consistent with the method name for it to start with a verb in Form as well (i.e., naming it changeUserSentiment as opposed to just userSentiment). What do you think?

File: src/components/Form/Form.tsx [hunk 3]
```diff
@@ -1 +1,36 @@
...
+  return(
+    <div className="form-container">
+      <section className="questionnaire">
+        <h2>how are you feeling today?</h2>
+        <article className="sentiment-selection">
+          <button className="happy-btn" name="1" onClick={assignSentiment}></button>
+          <button className="neutral-btn" name="0" onClick={assignSentiment}></button>
+          <button className="sad-btn" name="-1" onClick={assignSentiment}></button>
+        </article>
+        <article className="wellnews-synopsis">
```
Comment (nzambonivergara) [inline src/components/Form/Form.tsx:25]: Once we turn this into a button, should this be its own component that appears across all routes?

**Review by nzambonivergara — CHANGES_REQUESTED** [review]
Thanks for building the Form component! I added a couple of comments, let me know if you have any questions.

File: src/components/Form/Form.tsx [hunk 4]
```diff
@@ -1 +1,36 @@
...
+
+
+interface userSentimentProps {
+  userSentiment: (newSentiment: number) => void
+}
+
+const Form: React.FC<userSentimentProps> = props => {
+  const assignSentiment = (event: React.MouseEvent<HTMLButtonElement>) => {
+      event.preventDefault();
+      const button: HTMLButtonElement = event.currentTarget;
```
Comment (anthony-iacono) [inline src/components/Form/Form.tsx:12]: Nice use of `event.currentTarget`! Hadn't seen this before.

File: src/components/Form/Form.tsx [hunk 5]
```diff
@@ -1 +1,36 @@
...
+      props.userSentiment(Number(button.name));
+  }
+
+  return(
+    <div className="form-container">
+      <section className="questionnaire">
+        <h2>how are you feeling today?</h2>
+        <article className="sentiment-selection">
```
Comment (anthony-iacono) [inline src/components/Form/Form.tsx:20]: A `<div>` might be more appropriate here in lieu of an `<article>`. MDN on `<article>`: "a forum post, a magazine or newspaper article, or a blog entry, a product card, a user-submitted comment, an interactive widget or gadget, or any other independent item of content." The `<h2>` and the `<article>`'s children are intimately related, and probably not independent items of content.

File: src/components/Form/Form.tsx (hunk 3 above)
Comment (anthony-iacono) [reply src/components/Form/Form.tsx:25]: Good idea! Perhaps with a different titles, like 'Why we ask?' on the quiz and 'How we sort your feed' on the news feed.

**Review by anthony-iacono — CHANGES_REQUESTED** [review]
Please let us know once you've finished making whatever changes you'd like to make based on our feedback, then we'll go ahead and merge :)

File: src/components/Form/Form.tsx (hunk 5 above)
Comment (nzambonivergara) [reply src/components/Form/Form.tsx:20]: I would consider wrapping the question in a `<form>`. Though it is only one question, it is an interactive portion where the user submits information (a sentiment). What do you think?

File: src/components/App/App.tsx (hunk 2 above)
Comment (najinl) [reply src/components/App/App.tsx:78]: Can do! I like this suggestion.

File: src/components/Card/Card.tsx (hunk 1 above)
Comment (najinl) [reply src/components/Card/Card.tsx:9]: Just reformatted the Form component to not include React.FC and received the following error from eslint : 
"Line 10:14:  Missing return type on function  @typescript-eslint/explicit-module-boundary-types"
This was not a warning we got before, do we want to ignore this eslint warning and/or uninstall?

File: src/components/App/App.tsx [hunk 6]
```diff
@@ -14,7 +15,9 @@ export interface Article {
 }
 
 interface State {
-  articles: Article[]
+  articles: Article[],
+  userSentiment: number,
+  initialSentimentSet: boolean
```
[the remark this answers is not in the document]
Comment (najinl) [reply src/components/App/App.tsx:20]: done.

File: src/components/App/App.tsx [hunk 7]
```diff
@@ -56,17 +59,24 @@ class App extends React.Component<Props, State> {
   constructor(props: Props) {
```
[the remark this answers is not in the document]
Comment (najinl) [reply src/components/App/App.tsx:59]: Done in Form.tsx

File: src/components/Form/Form.tsx (hunk 5 above)
Comment (najinl) [reply src/components/Form/Form.tsx:20]: I changed it to <form> although I wonder if it really is much of a form? But Anthony makes a great point about it not seeming like an article, so form certainly feels like the next best option!

File: src/components/Card/Card.tsx (hunk 1 above)
Comment (nzambonivergara) [reply src/components/Card/Card.tsx:9]: Would adding the return type of JSX.Element fix that error?

**Pull request merged**